tensorflow/python/ops/map_fn.py

```diff
@@ -193,7 +193,8 @@ def map_fn(fn,
 
     * `tf.sparse.SparseTensor(st.indices, fn(st.values), st.dense_shape)`
       (if the function is expressible as TensorFlow ops)
-    * `tf.sparse.SparseTensor(st.indices, tf.map_fn(fn, st.values), st.dense_shape)`
+    * `tf.sparse.SparseTensor(st.indices, tf.map_fn(fn, st.values),
+      st.dense_shape)`
```

Comment:
This needs reformat. Preferably, switch the entire list to use this format

```
* condition:
    ```python
    tf.sparse...
    ```
```

and make sure code in the documentation still looks like validly formatted Python code

Reply:
Sorry, still not sure how to reformat this...

Reply:
Reformatted.